charts/cert-manager/values.selfsigned.yaml.gotmpl

```diff
@@ -22,6 +22,12 @@ extraObjects:
       "helm.sh/hook": post-install,post-upgrade
       "helm.sh/hook-weight": "1"
   spec:
+    secretTemplate:
+      annotations:
+        reflector.v1.k8s.emberstack.com/reflection-allowed: "true"
+        reflector.v1.k8s.emberstack.com/reflection-allowed-namespaces: ""  # Control destination namespaces
```

Comment:
does `""` mean allow all namespaces? If so, maybe let's add a comment

Reply:
yes it does mean that :palm_up_hand: